src/services/auth.forked.services.js

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import {
+  CHANGELOG_PR_BRIDGE_API_BASE_URL,
+} from "../config/constants.js";
+
+import {
+  handleChangelogPRBridgeResponseErrors,
+} from "../utils/index.js";
+/**
+ * Get a file in a given path in a forked GitHub repository.
+ *
+ * @param {string} owner - The repository owner.
+ * @param {string} repo - The repository name.
+ * @returns {Promise<object>} - An object containing the file details.
+ * @throws {Error} - If an error occurs while fetching the file.
```

Comment:
```suggestion
 * Get the github app installation info from the forked repo
 *
 * @param {string} owner - The repository owner.
 * @param {string} repo - The repository name.
 * @returns {Promise<object>} - An object containing the app info.
 * @throws {Error} - If an error occurs while fetching the file.
```

Reply:
Fixed! My bad. I didn't check the documentation I put on that function correctly.